src/mlpack/core/cv/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Define the files we need to compile
+# Anything not in this list will not be compiled into mlpack.
+set(SOURCES
+)
```

Comment:
I don't have too much experience with cmake, and I was looking at src/mlpack/core/data/CMakeLists.txt as an example while writing src/mlpack/core/cv/CMakeLists.txt, since src/mlpack/core/data also has some directories. Probably we should fix src/mlpack/core/data/CMakeLists.txt too. I will fix src/mlpack/core/cv/CMakeLists.txt shortly.